docs/team/kaichao.md

```diff
@@ -25,7 +30,8 @@ Given below are my contributions to the project.
 * **Documentation**:
   * User Guide:
     * Converted the draft user guide content to markdown file format.
-    * Updated the documentation for `delete` feature.
+    * Update the documentation for `delete` feature.
   * Developer Guide:
     * Update NFR and Appendix section for Warenager.
+    * Added MSS for delete feature.
```

Comment:
Should we standardise past tense / present tense? ("Update" vs "Added")

Reply:
I think can standardize as past tense for things that are already done.